server/tests/src/test/resources/database/mariadbgalera.properties

```diff
@@ -0,0 +1,18 @@
+id.column.type=VARCHAR(255)
+data.column.type=VARBINARY(1000)
+timestamp.column.type=BIGINT
+segment.column.type=BIGINT
+database.mode=CONTAINER
+database.container.name=toughiq/mariadb-cluster
```

Comment:
`quay.io` if possible

Reply:
We don't have this image option on `quay.io` and it has the same problem that `quay.io/bitnami/mariadb`.
But don't worry about use those images outside quay.io because we can override this property and set another images

Reply:
Yep that's fine, see https://github.com/infinispan/infinispan/pull/9293/files#r649253313